e2e/specs/results-viewer.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from '@playwright/test';
+import { type Visit } from '@openmrs/esm-framework';
+import { generateRandomPatient, type Patient, startVisit, endVisit, deletePatient } from '../commands';
+import { test } from '../core';
+import { ResultsViewerPage, ChartPage } from '../pages';
+
+let patient: Patient;
+let visit: Visit;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+  visit = await startVisit(api, patient.uuid);
+});
+
+test('Record and edit test result', async ({ page }) => {
+  const chartPage = new ChartPage(page);
+  const resultsViewerPage = new ResultsViewerPage(page);
+
+  await test.step('When I visit the chart summary page', async () => {
+    await chartPage.goTo(patient.uuid);
+  });
+
+  await test.step('Then I launch the results viewer page', async () => {
+    await resultsViewerPage.goTo(patient.uuid);
+  });
+
+  await test.step('And I click on the `Clinical forms` button on the siderail', async () => {
+    await chartPage.page.getByLabel(/clinical forms/i).click();
```

Comment:
```suggestion
    await page.getByLabel(/clinical forms/i).click();
```

is there any reason why you used `chartPage` here?
It will work without that too because we are passing `result viewer` page here. So we didn't need to add `chartPage` here

Reply:
@Vijaykv5 , I needed to launch the forms workspace to access the clinical forms.
So I opted to the re-use this `formsTable = () => this.page.getByRole('table', { name: /forms/i });` in `chartPage` which is by design a parent page.
It works if I introduce `formsTable = () => this.page.getByRole('table', { name: /forms/i });` in `resultsViewerPage` but then, we don't have a clinical forms table directly in `results viewer`.

I initially reverted this, but when I further understood the layout design, I realized I may have fallen out of design. 
Let me know what you think, please.

Reply:
@chimanwadike Try to get clinical forms table directly in results viewer. I think in updated commits, there reside some issue that's why the E2E tests failing?
Have you tested it locally?

Reply:
@Vijaykv5 I have pushed this change

Reply:
@Vijaykv5 Sorry I effected the changes and didn't answer questions. Your review was probably based on the E2E tests failure.
E2E test failure is already a known problem (flagged) for about 2 days now and it's been investigated by @jayasanka-sack. It's not connected to this commit and it works locally